src/main/resources/application.yaml

```
@@ -90,7 +90,18 @@ camundaGroups:
     groupId: civil
     accessControl: user
     s2sServiceNames:
-      - civil-service
+      - civil-service 
+  Privatelaw:
```

Making sure its small case, also aligning the name to make it easy
```suggestion
  prl:
```